bootstrap.md

```diff
@@ -1,5 +1,10 @@
 # Filecoin Bootstrapping Routine
 
+
+{{% notice warning %}}
+Outdated. The link that got you here should probably link to [`networking.md`](./networking.md) instead.
+{{% /notice %}}
+
```

Comment:
I'd prefer we delete the content below that we know is outdated/duplicated. People already familiar with the spec structure may continue to come here but miss this message, leading to confusion.

Reply:
wasn't sure about it and there is still stuff, that isn't fully moved yet, but I agree and will probably remove the file before bumping to merge it.